apis/cloudidentity/v1beta1/membership_types.go

```diff
@@ -32,6 +32,7 @@ type CloudIdentityMembershipSpec struct {
 	GroupRef GroupRef `json:"groupRef"`
 
 	// Immutable. The `EntityKey` of the member. Either `member_key` or `preferred_member_key` must be set when calling MembershipsService.CreateMembership but not both; both shall be set when returned.
```

Comment:
The comment literally said that this field should be set... isn't this contradicting with the output-only clarification?